zaza/openstack/charm_tests/test_utils.py

```diff
@@ -107,7 +107,7 @@ def resource_cleanup(cls):
     @classmethod
     def tearDown(cls):
         """Run teardown for test class."""
-        if cls.run_resource_cleanup:
+        if getattr(cls, 'run_resource_cleanup', False):
```

Comment:
I probably missed something, why not to declare it as a class attribute? That would avoid using a check each time we want to use it.

```
class OpenStackBaseTest(unittest.TestCase):
    run_resource_cleanup = False
    ...
```